lib_trades/data/gm4_trades/tags/functions/setup_trader.json

```diff
@@ -0,0 +1,5 @@
+{
+    "values": [
+        "gm4_trades-1.1:wandering_trader/modify"
```

Comment:
you need to verify the load.status before directly calling this function

Reply:
you also need to bump the minor load score for lib_trades since you made a change. Otherwise the libraries can't force priority

Reply:
> you need to verify the load.status before directly calling this function

What do you mean by this?

Reply:
You need an intermediate function that checks if the load scores are correct (major and minor are 1), before just calling the `modify` function. The reason a function tag is used is so the version checking is done in the library and not in the module that requires it. For example, if you had a `v1.2 lib_trades` and a `v1.1 lib_trades` they'd both run the `modify` function since there's no version checking, causing duplicate and possibly non-working function calls